### PR TITLE
Adds the Tesla Gauntlet

### PR DESCRIPTION
**Pull request by Alberyk**
The Tesla Gauntlet is a traitor item that allows this user to fire tesla lightning bolts every 30 seconds by clicking somewhere at the map with harm intent. Traitors can get it for ten telecrystals.

### CODE REVIEW

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 1]
```diff
@@ -315,3 +314,50 @@
...
+		to_chat(user, SPAN_WARNING("\The [src] is still recharging."))
+		return
+
+	if(user.a_intent == I_HURT)
+		var/turf/T = get_turf(user)
+		user.visible_message(SPAN_DANGER("\The [user] crackles with energy!"))
+		var/obj/item/projectile/beam/tesla/LE = new (T)
+		LE.muzzle_type = /obj/effect/projectile/muzzle/tesla
+		LE.tracer_type = /obj/effect/projectile/tracer/tesla
+		LE.impact_type = /obj/effect/projectile/impact/tesla
```
Comment (Geevies) [inline code/modules/clothing/gloves/miscellaneous.dm:345]: is it possible to turn this into a new projectile to not have to define this here

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 2]
```diff
@@ -315,3 +314,50 @@
 	. = ..()
 	if(mounted)
 		mounted.switch_firemodes()
+
+/obj/item/clothing/gloves/tesla
+	name = "tesla gauntlet"
+	desc = "A weaponized gauntlet capable of firing lightning bolts."
+	desc_fluff = "A tesla-based weapon created by the People's Republic of Adhomai as part of their Tesla Brigade program. Because of its long recharge time, the gauntlet is commonly \
+	used as an ancillary weapon."
+	icon_state = "tesla_glove_on"
```
Comment (Geevies) [inline code/modules/clothing/gloves/miscellaneous.dm:323]: can this be made a contained sprite instead?

Comment (Alberyk) [reply code/modules/clothing/gloves/miscellaneous.dm:323]: The gloves dmi is not a giant clusterfuck like the guns or the suit ones were, don't really think it is really needed here.

**Comment by Spacemanspark** [comment]
I see someone enjoys half life alyx

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**